v1.0/frontend/SaralApp/src/modules/ScannedDetails/MarksHeaderTable.js

```diff
@@ -80,7 +80,7 @@ const styles = {
         width: '100%',
         color: AppTheme.BLACK,
         fontWeight: 'bold',
-        letterSpacing: 1,
+        // letterSpacing: 1,
```

Comment:
@arman-tdev / @apurvabisht97 , any specific reason for removing letterSpacing: 1, ?